presto-common/src/test/java/com/facebook/presto/common/TestPage.java

```diff
@@ -246,6 +246,46 @@ public void testReplaceColumn_WrongNumberOfRows()
         page.replaceColumn(1, newBlock);
     }
 
+    @Test(expectedExceptions = IllegalArgumentException.class)
+    public void testPrependColumn_WrongNumberOfRows()
```

Comment:
nit: by convention we use camelCase. 
```suggestion
    public void testPrependColumnWrongNumberOfRows()
```